README.md

```diff
@@ -68,7 +68,20 @@ As PagerDuty itself does not provide finer granularity for webhooks than service
 To add a new alert investigation:
 
 - run `make bootstrap-investigation` to generate boilerplate code in `pkg/investigations` (This creates the corresponding folder & .go file, and also appends the investigation to the `availableInvestigations` interface in `registry.go`.).
-- investigation.Resources contain initialized clients for the clusters aws environment, ocm and more. See [Integrations](#integrations)
+- The `Run` method of your investigation receives a `ResourceBuilder`. Use its `With...` methods to request the resources your investigation needs, then call `Build()` to get a `Resources` struct containing them. The builder automatically handles dependencies between resources (e.g., requesting an AWS client will also initialize the cluster object). For example:
```

Comment:
I think we forgot to update the bootstrap:
https://github.com/openshift/configuration-anomaly-detection/blob/main/hack/bootstrap-investigation.sh

Reply:
Updated - I didn't know we had some scaffolding for investigations :partying_face: